### ISSUE TITLE
fastq not compatible for downstream analyses unless first line is removed

### ISSUE THREAD

**Issue by cdebuck** [nanoporetech/bonito#274]
I've been using bonito version 0.5.3 and I noticed that it prints the following line to the fastq output when it has used a model for the first time:
`[downloaded dna_r9.4.1_e8_sup@v3.3.zip]`

If I want to use the fastq output for further analyses I first have to remove this line, which I do find a bit inconvenient. I think it would be nice if this line would be reported to a log file or to another place instead of the fastq output.

**Comment by iiSeymour** [comment]
Good catch @cdebuck this is fixed in https://github.com/nanoporetech/bonito/commit/f0827d931df550eded9d87744df62ba718db9af6.
